databricks_cli/pipelines/api.py

```diff
@@ -130,25 +130,6 @@ def _get_hashed_path(path):
             path = '{}/{}.{}'.format(base_pipelines_dir, file_hash, extension)
         return path
 
-    @staticmethod
```

Comment:
This is for the
> Stop double-passing credentials to the Delta Pipelines service
part of the change. This method pulled CLI credentials from the config file so they could be included in the body of the request. Now that we don't need to include them in the body, we don't need this method. Arul wrote it nicely so that we could just whack this method once we fixed the credentials server-side situation.

Reply:
ok